dashboard/app/controllers/aichat_controller.rb

```diff
@@ -117,6 +121,40 @@ def log_chat_event
     render(status: :ok, json: response_body)
   end
 
+  # params are eventId: number, feedback?: TeacherFeedback
```

Comment:
Can we be more explicit about what TeacherFeedback is?
`feedback?:   'clean_disagree' | 'profanity_agree' |  'profanity_disagree'`

Reply:
Also, as a follow-up, might be nice to store these values in shared constants if they're referenced in the backend as well.

Reply:
Updated comment and Noted follow-up in description